package.json

```diff
@@ -42,6 +42,13 @@
     "typescript": "2.0.3"
   },
   "devDependencies": {
+    "@types/fs-extra": "0.0.33",
+    "@types/handlebars": "^4.0.31",
+    "@types/highlight.js": "^9.1.8",
+    "@types/lodash": "^4.14.37",
+    "@types/marked": "0.0.28",
+    "@types/minimatch": "^2.0.29",
+    "@types/shelljs": "^0.3.32",
```

Comment:
That's understandable. The issue with it is that you might accidentally expose one or the other at some point and forget to move them around. I think putting them in the same place as the real dependency is the best approach to avoid unfortunate side-effects.